test/controllers/NiphlNumberControllerSpec.scala

```diff
@@ -56,6 +60,44 @@ class NiphlNumberControllerSpec extends SpecBase with NiphlNumberGenerator {
 
     }
 
+    "must return OK and the full view for a GET" in {
+
+      val fullNiphlNumberController = new NiphlNumberController(
```

Comment:
If we can put this full controller next to the other empty one at the top, that'd be nice, but it'm not fussed if you're happy.

Reply:
i was going to day because it is only used in one test keep it within that scope?

Reply:
Fine by me!